WebIDL/ecmascript-binding/window-named-properties-object.html

```diff
@@ -0,0 +1,282 @@
+<!doctype html>
+<meta charset="utf-8">
+<title>Internal methods of Window's named properties object</title>
+<link rel="help" href="https://heycam.github.io/webidl/#named-properties-object">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<body>
+<script>
+function looseModeSet(base, key, value) { base[key] = value; }
```

Comment:
Nit: traditionally we call it "sloppy mode" not "loose mode".

Reply:
Nice, we call it "sloppy" in JSC as well. d7eef58da0